rapid7_insightidr/plugin.spec.yaml

```diff
@@ -124,6 +124,188 @@ types:
       description: Indicates whether this threat has been published
       type: boolean
       required: false
+  link:
+    rel:
+      title: Relation
+      description: Relation
+      type: string
+    href:
+      title: Href
+      description: Href
+      type: string
+  eventData:
+    oaState:
+      title: OA State
+      type: string
+      description: OA state
+      required: false
+      lastAsSignatureAge:
+        title: Last As Signature Age
+        type: string
+        description: Last as signature age
+        required: false
+      productStatus:
+        title: Product Status
+        type: string
+        description: Product status
+        required: false
+      assignatureVersion:
+        title: As Signature Version
+        type: string
+        description: As signature version
+        required: false
+      ioavState:
+        title: IOAV State
+        type: string
+        description: IOAV state
+        required: false
+      asSignatureCreationTime:
+        title: As Signature Creation Time
+        type: string
+        description: As signature creation time
+        required: false
+      lastFullScanSource:
+        title: Last Full Scan Source
+        type: string
+        description: Last full scan source
+        required: false
+      nriSignatureVersion:
+        title: NRI Signature Version
+        type: string
+        description: NRI Signature Version
+        required: false
+      lastFullScanStartTime:
+        title: Last Full Scan Start Time
+        type: string
+        description: Last full scan start time
+        required: false
+      engineVersion:
+        title: Engine Version
+        type: string
+        description: Engine version
+        required: false
+      bmState:
+        title: BMS State
+        type: string
+        description: BMS state
+        required: false
+      lastQuickScanSource:
+        title: Last Quick Scan Source
+        type: string
+        description: Last quick scan source
+        required: false
+      lastFullScanEndTime:
+        title: Last Full Scan End Time
+        type: string
+        description: Last full scan end time
+        required: false
+      productName:
+        title: Product Name
+        type: string
+        description: Product name
+        required: false
+      rtpState:
+        title: RTP State
+        type: string
+        description: RTP state
+        required: false
+      platformVersion:
+        title: Platform Version
+        type: string
+        description: Platform version
+        required: false
+      lastQuickScanAge:
+        title: Last Quick Scan Age
+        type: string
+        description: Last quick scan age
+        required: false
+      data:
+        title: Data
+        type: "[]object"
```

Comment:
Is this typable? `object` is not very usable for users

Reply:
I will test again, but for different events there are returned different data. So it will be hard to change this to type. Maybe I will find few fields that are the same.